packages/editor/src/components/widgets/table/cell/ExpandableCell.tsx

```diff
@@ -22,13 +24,19 @@ export function ExpandableCell<TData>({ cell, isLoaded, loadChildren, isUnknown,
   return (
     <div className='row-expand' style={{ paddingLeft: `${row.depth}rem` }} title={title}>
       {row.getCanExpand() ? (
-        <Button
-          icon={IvyIcons.AngleDown}
-          className='row-expand-button'
-          aria-label={row.getIsExpanded() ? 'Collapse row' : 'Expand row'}
-          data-state={row.getIsExpanded() ? 'expanded' : 'collapsed'}
-          {...{ onClick: row.getToggleExpandedHandler() }}
-        />
+        <>
+          <Button
+            icon={IvyIcons.AngleDown}
+            className='row-expand-button'
+            aria-label={row.getIsExpanded() ? 'Collapse row' : 'Expand row'}
+            data-state={row.getIsExpanded() ? 'expanded' : 'collapsed'}
+            {...{ onClick: row.getToggleExpandedHandler() }}
+          />
+          {additionalIcon?.length == 0 && additionalIcon !== undefined && '🔵'}
+          {additionalIcon?.length !== 0 && additionalIcon !== undefined && (
+            <IvyIcon icon={IvyIcons[additionalIcon as keyof typeof IvyIcons]} />
+          )}
```

Comment:
No! You write typescript code and not javascript!
If your additionalIcon (which I would still rename to just icon) would be of type IvyIcons in the first place you didn't have to do all this ugly typechecking stuff!
Just render the icon if it is not undefined.

And I would not render this blue dot here, because it would not match with the other code. You already show the type of the content object right? I think this is enough for the start to give the user the hint if he can click on something or not. We can still improve this after we have more icons and restyle this tree tables